webapp/src/dogma/common/components/Navbar.tsx

```diff
@@ -180,8 +180,9 @@ export const Navbar = () => {
                 <Avatar name={user.login} size="sm" />
               </MenuButton>
               <MenuList>
-                <MenuItem>Application tokens</MenuItem>
-                <MenuItem>Add ...</MenuItem>
+                <MenuItem>
+                  <RouteLink href="/app/settings/tokens">Application tokens</RouteLink>
```

Comment:
Should we use `ChakraLink`?

Reply:
`ChakraLink` shows an underline on hover. We may not need that on the nav menu.

Reply:
👍 Thanks for the explanation.